docs/pages/x/api/charts/spark-line-chart.json

```diff
@@ -74,6 +74,12 @@
       "default": "BarElementPath",
       "class": null
     },
+    {
+      "name": "barLabel",
```

Comment:
It looks like we are missing the change in `SparklineChart` to make the `barLabel` prop work.
Or should it be removed from that component props if we don't want it there? 🤔

Reply:
I'm in favor of removing it from the props typing. Not very usefull for scatter bar plot, and not sure we want to keep this feature when reworking the sparkline to make it more performant.

NB: José, has you can see having the scripts result in the PR help for the review process 😉